pkg/cmd/kafka/topic/create/create.go

```diff
@@ -203,7 +202,7 @@ func runCmd(opts *options) error {
 }
 
 func runInteractivePrompt(opts *options) (err error) {
-	validator := topicutil.Validator{
+	validator := topiccmdutil.Validator{
```

Comment:
I'd say eventually, validators wont be in the util package as they are not util methods, again that can happen during the SDK creation.